### DISCUSSION TITLE
Take the 2021 Doks Survey

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#534] [General]
What do you use Doks for? How do you rate Doks? What Doks future features are important to you? We want to hear from you! The survey should take just 5 minutes to complete. No personal data is collected. Thanks for your participation.

[Take the 2021 Doks Survey →](https://docs.google.com/forms/d/e/1FAIpQLSe0DItAFv2cNSC0S_L6DQBw9ofB0ogYvZqeoWHHRX-Aal-5zQ/viewform?usp=sf_link)

**Comment by nk9** [comment]
A bit late to the party, but I'm using Doks for a podcast site. I'm excited about the solid foundation with Open Graph/SEO and the search functionality.

Things I'm particularly keen to see included:
- More hooks to add custom code (SCSS, JS, head content, etc).
- Image lightbox to examine and flip through images on a page, for example [PhotoSwipe](https://photoswipe.com).
- Make sure search can easily be extended to other content types
- A bit more detail on the docs about how to set up and use the theme. For example, this is all a bit terse, and doesn't explain how to use the theme in child mode:

![image](https://user-images.githubusercontent.com/3646730/148665449-f57539d5-73bc-4511-9ec9-f0eb5f9e62c8.png)

Thanks for all the work that has gone into Doks so far!

**Reply by h-enk** [reply]
Thanks for your input! I will process it and put it on the list

**Comment by wphromma** [comment]
Even though I am new to Hugo and Doks, while I was searching for a good theme for documentation of our Gitlab CI templates, I found this doc theme is pretty good. The doc searching is the best and the look-and-feel of the theme is beatiful.

I just adapt it to work dynamically. Every page is created during the deployment (from the generator script) and deployed to Gitlab Pages. It's a little tough at first cause I faced many issues but finally manage to resolve them.

Thank you so much for your great work and keep doing it !!

![screencapture-wphromma-pages-agodadev-io-ci-template-book-docs-ansible-ansible-lint-2022-01-09-20_38_12](https://user-images.githubusercontent.com/72568309/148684685-dc038adf-8a9f-4732-8ab3-d6a21f8f62fb.png)

**Reply by h-enk** [reply]
Thanks, great to hear!

> I just adapt it to work dynamically. Every page is created during the deployment (from the generator script) and deployed to Gitlab Pages. It's a little tough at first cause I faced many issues but finally manage to resolve them.

What issues were you facing? And, for the benefit of others, what script did work for you (something like https://github.com/h-enk/doks/discussions/184)?
